backend/src/test/java/com/allog/dallog/category/service/CategoryServiceTest.java

```diff
@@ -0,0 +1,47 @@
+package com.allog.dallog.category.service;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import com.allog.dallog.category.dto.request.CategoryCreateRequest;
+import com.allog.dallog.category.exception.InvalidCategoryException;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+
+@SpringBootTest
+class CategoryServiceTest {
+
+    @Autowired
+    private CategoryService categoryService;
+
+    @DisplayName("새로운 카테고리를 생성한다.")
+    @Test
+    void 새로운_카테고리를_생성한다() {
+        // given
+        String name = "BE 공식일정";
+        CategoryCreateRequest request = new CategoryCreateRequest(name);
+
+        //when
+        Long id = categoryService.save(request);
+
+        // then
+        assertThat(id).isNotNull();
```

Comment:
```suggestion
        // given
        String name = "BE 공식일정";
        CategoryCreateRequest request = new CategoryCreateRequest(name);

        // when
        Long id = categoryService.save(request);

        // then
        assertThat(id).isNotNull();
```

(많이 사소하지만 😅) 공백을 하나 추가할까요?